aap_chatbot/src/useChatbot/useChatbot.ts

```diff
@@ -60,12 +60,11 @@ export const inDebugMode = () => {
 
 export const isStreamingSupported = () => {
   // For making streaming mode debug easier.
-  //   if (!import.meta.env.PROD && import.meta.env.MODE.includes("stream")) {
```

Comment:
This change was added for hack-a-thon version. We need to wait until we can  fully enable streaming with AAP-UI integration. The original logic disables streaming by default.